itests/kit/node_miner.go

```diff
@@ -224,3 +224,36 @@ func (tm *TestMiner) SectorsListNonGenesis(ctx context.Context) ([]abi.SectorNum
 
 	return l[tm.PresealSectors:], nil
 }
+
+type SchedInfo struct {
+	CallToWork   struct{}
+	EarlyRet     interface{}
+	ReturnedWork interface{}
+	SchedInfo    struct {
```

Comment:
Is this intentional? Might wanna have a different name for this struct object than the higher level type?

Reply:
This is a horrible hack extracted from a different test; Technically the test can depend on the real struct, but that's a tiny bit tricky now

Reply:
Oh it's not easy to reuse that struct..

See https://github.com/filecoin-project/lotus/blob/master/storage/sealer/manager.go#L1225-L1239

This code was written in the days where we had big plans to make scheduling / sealing code easily swappable, but that has resulted in this vague loose diagnostic interface which tries hard to not commit to a schema. Probably should be fixed one day, but maybe out of scope for this PR.

Reply:
Can you add a comment referencing the place in code this is being replicated from? In the event there are changes made to the struct in manager.go, things will need to change here too?
Also probably a TODO to use a single definition